src/subscription/data/sagas.js

```diff
@@ -10,7 +10,7 @@ import { fetchSubscriptionDetails, submitPayment } from './details/actions';
 import { handleFetchSubscriptionDetails, handleSubmitPayment } from './details/sagas';
 
 export default function* saga() {
-  yield takeEvery(fetchSubscriptionClientSecret.TRIGGER, handleFetchClientSecret);
-  yield takeEvery(fetchSubscriptionDetails.TRIGGER, handleFetchSubscriptionDetails);
-  yield takeEvery(submitPayment.TRIGGER, handleSubmitPayment);
+  yield takeLatest(submitPayment.TRIGGER, handleSubmitPayment);
```

Comment:
Nice switch to `takeLatest()`, since we only need the latest value and not every single value

Reply:
Thanks